### PR TITLE
Added Code of Conduct transparency reports

### PR DESCRIPTION
**Pull request by tracykteal**
Adding directory for Code of Conduct transparency reports, and report for November from the Executive Council.

### CODE REVIEW

**Comment by k8hertweck** [comment]
Thanks for adding this information, Tracy.

**Comment by tracykteal** [comment]
Reviewed by @elliewix @ethanwhite and should be ready to merge

File: code-of-conduct-transparency-reports/2018-11-09-coc-transparency-report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+**Resolutions:**
+
+Of the two potential Code of Conduct matters:
+- Two were resolved post-workshop.
+
+## Summary of Reported Incidents
+
+- An instructor was reported for using language that was not welcoming and/or inclusive in a workshop. The Code of Conduct committee resolved this incident by issuing a reprimand. The instructor was responsive to feedback, and the CoC committee will be monitoring the survey results from the instructor’s upcoming workshops to verify that such behavior does not occur again.
+- An instructor was reported for using verbal comments which had the effect of excluding people on the basis of membership of a specific group in a workshop. The Code of Conduct committee resolved this issue by issuing a reprimand, requiring the reported individual to take a two month Termed Suspension, and that they receive and be responsive to coaching from staff on culturally responsive teaching.
```
Comment (ethanwhite) [inline code-of-conduct-transparency-reports/2018-11-09-coc-transparency-report.md]:
```suggestion
- An instructor was reported for using verbal comments which had the effect of excluding people on the basis of membership of a specific group in a workshop. The Code of Conduct committee resolved this issue by issuing a reprimand, requiring the reported individual to take a two month Termed Suspension, and requiring that the individual receive and be responsive to coaching from staff on culturally responsive teaching.
```

**Comment by tracykteal** [comment]
Thanks, accepted those edits.

**Pull request merged**